### ISSUE TITLE
Works with my v3 but does not seem to work with v2 or outdoor cams

### ISSUE THREAD

**Issue by pbarone** [mrlt8/docker-wyze-bridge#61]
For my v2 I see this error in the logs

2021/08/07 22:49:36 [Kitchen Cam] Camera firmare may be incompatible

**Comment by mrlt8** [comment]
Can you post the firmware version of your V2?

**Comment by pbarone** [comment]
4.9.7.798

**Comment by Vansmak** [comment]
same here

**Comment by sweichbr** [comment]
And does anyone know how to downgrade the firmware, if that is the cause of the issue?  I know how to flash firmware manually, but no idea if older firmwares (other than the official RSTP) are available online.

**Comment by Vansmak** [comment]
> 
> 
> And does anyone know how to downgrade the firmware, if that is the cause of the issue? I know how to flash firmware manually, but no idea if older firmwares (other than the official RSTP) are available online.

you mean to rollback?  Because the latest firmware is not working. Interestngly the same thing happened with tinycam, you hade to use the beta of tinycam after the last wyze firmware

**Comment by EliotHertenstein** [comment]
Same issue here with my v2

**Comment by sweichbr** [comment]
yeah - exactly, there is a note that the latest firmware is incompatible.  If someone can locate the firmware, it can be flashed per: https://support.wyze.com/hc/en-us/articles/360026245231-Wyze-Cam-RTSP

**Comment by EliotHertenstein** [comment]
Looks like you can downgrade through the app

https://forums.wyzecam.com/t/firmware-downgrade/19154

**Comment by sweichbr** [comment]
Eliot: 

Looks like that feature is no longer available in the current app release.

I did find this: https://support.wyze.com/hc/en-us/articles/360041676711-How-do-I-revert-my-webcam-back-to-a-Wyze-Cam-

All the firmware are linked in the version number.

Now to determine which one works with the RTSP bridge.

**Comment by mckochan** [comment]
I can confirm it works with 4.9.6.241 on my v2.

**Comment by jmacul2** [comment]
Wyze is moving off of TUKT and using WebRTC so I would assume this will stop working all together soon.  In any case, WebRTC is a way better solution.  I am sure someone will find an API entry point to grap the WebRTC url soon enough.
